pkgs/shells/zsh/zsh-edit/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-edit";
+  version = "unstable-2022-05-05";
+
+  src = fetchFromGitHub {
+    owner = "marlonrichert";
+    repo = "zsh-edit";
+    rev = "4a8fa599792b6d52eadbb3921880a40872013d28";
+    sha256 = "PI4nvzB/F0mHlc0UZJdD49vjzB6pXhhJYNTSmBhY8iU=";
+  };
+
+  strictDeps = true;
```

Comment:
it looks like there is a TODO to make this the default, is there some reason that we are building without `strictDeps` or need `strictDeps` to make this derivation work?

Reply:
no, strictDeps just makes sure that nativeBuildInputs and buildInputs do not mix. Here it should be irrelevant.